src/event/events.js

```diff
@@ -94,18 +94,22 @@ const HTML5_EVENTS: { [event: string]: string } = {
 
 const CUSTOM_EVENTS: { [event: string]: string } = {
   /**
-   * Fires when the video track has been changed
+   * Fires when the active video track has been changed
    */
   VIDEO_TRACK_CHANGED: 'videotrackchanged',
   /**
-   * Fires when the audio track has been changed
+   * Fires when the active audio track has been changed
    */
   AUDIO_TRACK_CHANGED: 'audiotrackchanged',
   /**
-   * Fires when the text track has been changed
+   * Fires when the active text track has been changed
    */
   TEXT_TRACK_CHANGED: 'texttrackchanged',
   /**
+   * Fires when the player tracks have been changed
+   */
+  TRACKS_CHANGED: 'trackschanged',
```

Comment:
@dan-ziv in high level this might be more accurate, as, for example, in live manifests the tracks can change during the live stream life time.
For example, Shaka uses TracksChangedEvent, while HLS.JS uses discrete audio and text track event AUDIO_TRACKS_UPDATED and SUBTITLE_TRACKS_UPDATED